### ISSUE TITLE
Doesn't create web root

### ISSUE THREAD

**Issue by goetzk** [aalaesar/install_nextcloud#47]
Hi,
The role doesn't appear to create webroot, it should probably done just before the extraction step.
```
TASK [aalaesar.install_nextcloud : Download & extract Nextcloud to /tmp.] *******************************************************************************************************************************************************
changed: [192.168.1.121]

TASK [aalaesar.install_nextcloud : [NC-DL] - Move extracted files to /srv/nextcloud/webroot.] ***********************************************************************************************************************************
fatal: [192.168.1.121]: FAILED! => {"changed": true, "cmd": ["cp", "-r", "/tmp/nextcloud/.", "/srv/nextcloud/webroot/"], "delta": "0:00:00.007128", "end": "2018-10-12 06:00:27.131434", "msg": "non-zero return code", "rc": 1, "start": "2018-10-12 06:00:27.124306", "stderr": "cp: cannot create directory '/srv/nextcloud/webroot/': No such file or directory", "stderr_lines": ["cp: cannot create directory '/srv/nextcloud/webroot/': No such file or directory"], "stdout": "", "stdout_lines": []}

RUNNING HANDLER [aalaesar.install_nextcloud : start apache] *********************************************************************************************************************************************************************
```

**Comment by aalaesar** [comment]
hi @goetzk 
given the small amount of info,
I suppose the issue is that `/srv/nextcloud` did not existed when you run the playbook calling the role.
(because most of the tests don't fails on this task when using /opt/nextcloud as webroot)
can you confirm it works when the basename exits ?
If that's the case; I'll add a task to either check it exists and/or creates it.
Regards
Aal.

**Comment by egmont1227** [comment]
This is fixed in PR #59  (tested with nextcloud_webroot: /srv/nexcloud/webroot)

**Comment by goetzk** [comment]
thanks
